src/Simplex/Chat.hs

```diff
@@ -127,12 +127,17 @@ logCfg :: LogConfig
 logCfg = LogConfig {lc_file = Nothing, lc_stderr = True}
 
 simplexChat :: WithTerminal t => ChatConfig -> ChatOpts -> t -> IO ()
-simplexChat cfg opts t =
-  -- setLogLevel LogInfo -- LogError
-  -- withGlobalLogging logCfg $ do
-  initializeNotifications
-    >>= newChatController cfg opts t
-    >>= runSimplexChat
+simplexChat cfg opts@ChatOpts {agentLogging} t =
+  if agentLogging
+    then do
+      setLogLevel LogInfo -- LogError
+      withGlobalLogging logCfg initRun
+    else initRun
```

Comment:
```suggestion
simplexChat cfg opts@ChatOpts {agentLogging} t
  | agentLogging = do
    setLogLevel LogInfo -- LogError
    withGlobalLogging logCfg initRun
  otherwise = initRun
```